__tests__/app.test.js

```diff
@@ -34,4 +35,23 @@ describe('api/topics', () => {
     .get('/api/nonsense')
     .expect(404)
   })
+})
+
+describe('api/', () => {
+  test('GET 200: returns a JSON object of all APIs', () => {
+    return request(app)
+    .get('/api')
+    .expect(200).then(({body})=> {
+      const apisObject = body.apis
+
+      expect(typeof apisObject).toBe('object')
+      expect(Array.isArray(apisObject)).toBe(false)
+      expect(apiEndpointsFile).toEqual(apisObject)
```

Comment:
Lovely stuff, strictly speaking you don't need line 47 and 48 as line 49 covers everything in one go